docs/documentation/full-text/filtering.mdx

```diff
@@ -2,96 +2,80 @@
+USING bm25(id, description, rating, created_at)
+WITH (key_field = 'id');
 ```
 
-## Boolean Filter
-
-Boolean fields can be filtered with `true` or `false`.
-
-```sql
-SELECT description, rating, category
-FROM mock_items
-WHERE description @@@ 'shoes' AND in_stock @@@ 'true';
-```
-
-## Range Filter
-
-The inclusive `[]` and exclusive `{}` bounds can be used for range filtering.
-
-```sql
--- Numeric
-SELECT description, rating, category
-FROM mock_items
-WHERE description @@@ 'shoes' AND rating @@@ '[1 TO 4]';
-
--- Datetime
-SELECT description, rating, category
-FROM mock_items
-WHERE description @@@ 'shoes' AND created_at @@@ '[2020-01-31T00:00:00Z TO 2024-01-31T00:00:00Z]';
-```
-
-Range filters can also be applied over text fields. The following query finds all documents where
-`description` contains a token that is lexicographically between `book` and `camera` (i.e. would come between
-`book` and `camera` in a dictionary).
-
-```sql
-SELECT description, rating, category
-FROM mock_items
-WHERE description @@@ '[book TO camera]';
-```
-
-## Set Filter
-
-The `IN` operator finds documents that match at least one element in a set.
-
-```sql
-SELECT description, rating, category
-FROM mock_items
-WHERE description @@@ 'shoes' AND rating @@@ 'IN [2 3 4]';
-```
+Filter pushdown is currently supported for the following combinations of types and operators:
+
+| Operator                                   | Left Operand Type | Right Operand Type | Example                    |
+| ------------------------------------------ | ----------------- | ------------------ | -------------------------- |
+| `=`, `<`, `>`, `<=`, `>=`, `<>`, `BETWEEN` | `int2`            | `int2`             | `WHERE rating = 2`         |
+|                                            | `int4`            | `int4`             |
+|                                            | `int8`            | `int8`             |
+|                                            | `int2`            | `int4`             |
+|                                            | `int2`            | `int8`             |
+|                                            | `int4`            | `int8`             |
+|                                            | `float4`          | `float4`           |
+|                                            | `float8`          | `float8`           |
+|                                            | `float4`          | `float8`           |
+|                                            | `date`            | `date`             |
+|                                            | `time`            | `time`             |
+|                                            | `timetz`          | `timetz`           |
+|                                            | `timestamp`       | `timestamp`        |
+|                                            | `timestamptz`     | `timestamptz`      |
+|                                            | `text`            | `text`             |
```

Comment:
Range queries do still require fast fields AFAIK: while it might not be worth mentioning for `numeric` types (since they are fast by default), it is still worth mentioned for text.